packages/main/src/ToolbarItem.ts

```diff
@@ -107,10 +132,27 @@ class ToolbarItem extends UI5Element {
 			},
 		};
 	}
+
+	constructor() {
+		super();
```

Comment:
since the constructor doesn't handle any arguments, this override can be omitted